### ISSUE TITLE
Error: Failed to obtain HOSTUUID from XE CLI

### ISSUE THREAD

**Issue by rainer042** [dalgibbard/citrix_xenserver_patcher#70]
Hi,

just tried patcher version 1.6.1 on a XenServer  6.2 as well as on 7.1. On both systems patcher.py is unable to get the host uuid and thus patching fails.  

It would be nice if this problem  could be fixed.

**Comment by dalgibbard** [comment]
This is a Xenserver issue where the host reports a hostname different to that of its current on-disk configuration. See: https://github.com/dalgibbard/citrix_xenserver_patcher/issues/64

Please let me know if this does/doesn't resolve the problem :)

**Comment by rainer042** [comment]
Thanks for the hint.  Since I upgraded my 6.2 installation to 7.1, I was only able to test on 7.1:

I tried the hostnamectl set-hostname fqdn command which did not do the job but had the unpleasant side effect that the hostname as printed by the hostname command is always the fqdn name.

What helped was to set the fqdn of the host in /etc/sysconfig/network by assigning the fqdn to the  HOSTNAME variable.

Thanks a lot
Rainer
